README.md

```diff
@@ -0,0 +1,27 @@
+### Repo Tracker
+
+### Installation & usage
+
+Once you have cloned the repo.
+
+first `npm install`.
+
+Then `npm run dev` to start application
+
+### Bugs
+
+1. The Back button does not functions as required.
+
+### Future features
+
+An additional Page for User information.
+
+### Wins & Challenges
+
+The Search form is functioning, and we are able fetch user information by their username.
+
+Once the Username has been entered, we able to retrieve all of their repository in a order list format.
+
+Then the user is able to click on any element within the list to retieve specific information regarding that particular repository.
+
+We are able to retieve the Repo_name, Fork_count and description and url from the GitHub Api.
```

Comment:
I like the repo - I was going to comment on the back button not working but I see you have it in bugs here.